### PR TITLE
Proper RTL support in weather forecast card and forcing LTR in dev panels

### PR DESCRIPTION
**Pull request by yosilevy**
My first PR so pardon any errors. I tried to exclude yarn.lock but couldn't.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @yosilevy, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#2405).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by yosilevy** [comment]
Signed CLA

**Comment by balloob** [comment]
Don't remove yarn.lock and include a screenshot before/after

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,5 @@ dist
 # Secrets
 .lokalise_token
 yarn-error.log
+
+yarn.lock
```
Comment (balloob) [inline .gitignore]: ????? remove this

File: src/panels/dev-mqtt/ha-panel-dev-mqtt.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ class HaPanelDevMqtt extends PolymerElement {
           padding: 24px 0 32px;
           max-width: 600px;
           margin: 0 auto;
+          direction: ltr;
```
Comment (balloob) [inline src/panels/dev-mqtt/ha-panel-dev-mqtt.js]: Why would this do anything?

**Comment by balloob** [comment]
I suggest you open a new PR, limit your change to 1 page, include screenshots, and don't mess with yarn.lock

**Comment by yosilevy** [comment]
Weather  - before - all messed up and degrees on wrong side of number...
weather-before

After:
weather-after

Dev panels

Devinfo - before - not readable since hebrew and english is not in the right order. Anyway all users would want it to be English since otherwise we need to manually switch to English to make any sense of it.
devinfo-before

After:
devinfo-after

Events - before - out of order and json can't be used.
events-before

After:
events-after

Services - before - hard to read service and entity names, JSON impossible...
services-before

After:
states-after

States - before - table not readable at all... no one can work like this
states-before

After:
states-after

Templates - before - you get the idea...
templates-before

After:
templates-after

Logbook - before - data in sentences is other way around and also very hard to read and follow
logbook-before

After:
logbook-after

MQTT - before - again - payload won't be readable, etc'...
mqtt-before

After - payload would be readable
mqtt-after

**Pull request closed without merging**